### ISSUE TITLE
support for neovim(null-ls)

### ISSUE THREAD

**Issue by shaeinst** [Riverside-Healthcare/djLint#184]
my [pull request-682 ](https://github.com/jose-elias-alvarez/null-ls.nvim/pull/682) for supporting djLint as formatter in [null-ls](https://github.com/jose-elias-alvarez/null-ls.nvim/) get merged. But i just found that formatting with djList also inserts "
**0 files were updated.**" line of code on current source file.

![image](https://user-images.githubusercontent.com/41078534/153906204-cf7a9abe-483c-4e7a-8096-e5710d66dcfd.png)

**Comment by christopherpickering** [comment]
Hi, Do you have an example command that causes a failure? I'm not sure how to debug your integration.

The quiet option just skips printing the diff, it doesn't suppress all output. Using stdin will surpress all output.

From the docs, this is how it should work for a "stdin" application:
```sh
echo "<div></div>" | djlint - --reformat
```

![image](https://user-images.githubusercontent.com/17788706/153907388-d76cee08-1b7a-42da-99fa-5047f2bd5ab6.png)

Maybe try something like this?

![image](https://user-images.githubusercontent.com/17788706/153907456-5be7220e-f19d-414d-8b06-29d3bc464b93.png)

Is there a Lua setting to use stdin instead of passing the file, or is it trying to pass the full file contents?

**Comment by shaeinst** [comment]
thanks. you're right. I'm going to make PR to solve this.
btw, do you have any plan to integrate djLint as linting for neovim (maybe using null-ls)?

**Comment by christopherpickering** [comment]
Cool, keep me posted! If you can't get stdin to work, we could look at making the ``--quiet`` option completely quiet.

I really only use sublime text, but feel free to add it as an integration. Do you mind doing a PR with a link to your tool for the https://djlint.com/docs/integrations/ page?

https://github.com/Riverside-Healthcare/djLint/blob/master/docs/src/docs/integrations.md

**Comment by christopherpickering** [comment]
Looks like you got this fixed. I'm going to close this issue, feel free to open another if you need help, or join our chat.
